please.sh

```diff
@@ -3453,6 +3453,7 @@ mention () { # <what, e.g. bug-fix, new-feature> <release-notes-item>
 		sed -i -e "/^## Changes since/{s/^/$quoted\n\n/;:1;n;b1}" \
 			"$relnotes"
 	else
+		search=$(echo $quoted | sed -r -e 's|.*Comes with \[(.* v).*|\1|')
```

Comment:
I think this breaks for the MSYS2 runtime upgrades... (see bd6aa98e43744625c0027c945726c30c9449e001 for an example outcome of the new feature).